packages/flutter_tools/test/integration.shard/hot_reload_test.dart

```diff
@@ -7,163 +7,120 @@ import 'dart:async';
 import 'package:file/file.dart';
 import 'package:flutter_tools/src/base/common.dart';
 import 'package:flutter_tools/src/base/file_system.dart';
-import 'package:vm_service/vm_service.dart';
 
 import '../src/common.dart';
 import 'test_data/hot_reload_project.dart';
 import 'test_driver.dart';
 import 'test_utils.dart';
 
+final HotReloadProject project = HotReloadProject();
+
 void main() {
   Directory tempDir;
-  final HotReloadProject _project = HotReloadProject();
-  FlutterRunTestDriver _flutter;
+  FlutterRunTestDriver flutter;
 
   setUp(() async {
     tempDir = createResolvedTempDirectorySync('hot_reload_test.');
-    await _project.setUpIn(tempDir);
-    _flutter = FlutterRunTestDriver(tempDir);
+    await project.setUpIn(tempDir);
+    flutter = FlutterRunTestDriver(tempDir);
   });
 
   tearDown(() async {
-    await _flutter?.stop();
+    await flutter?.stop();
     tryToDelete(tempDir);
   });
 
   test('hot reload works without error', () async {
```

Comment:
An advantage of using `testUsingContext` is that the test context can forbid some operations that would make tests flaky, like changing the cwd.

Reply:
Done

Reply:
Not seeing it =)

Reply:
Sorry, I commented again - the `flutter` wrapper uses globals so the change breaks the tests :(